### PR TITLE
DR-3061/tag new pages with aa

### PR DESCRIPTION
**Pull request by avertrees**
## Ticket:

- JIRA ticket [ tag new DCF pages with Adobe Analytics](https://newyorkpubliclibrary.atlassian.net/browse/DR-3061)

## This PR does the following:

- tags all pages with Adobe Analytics (I left collections/[:uuid] to just pass the url parameter into AA, regardless if it's a uuid or a slug)
- 
## How has this been tested?

locally with debugger

## Accessibility concerns or updates

-

### Checklist:

- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.
- [x] I have updated the CHANGELOG.md.

### CODE REVIEW

File: app/src/components/pageLayout/pageLayout.tsx [hunk 1]
```diff
@@ -34,9 +36,9 @@ const PageLayout = ({
     "asPath has been removed because the concept of as has been removed from the new router."
     https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration
     */
-    const route = pathname || "";
```
Comment (EdwinGuzman) [inline app/src/components/pageLayout/pageLayout.tsx:37]: I don't think the above comment is relevant anymore.

File: app/collections/lane/[slug]/page.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ export async function generateMetadata({
 }
 
 export default function Lane({ params }: LaneProps) {
+  const pageName = `dc|collections|lane|${params.slug}`;
```
Comment (EdwinGuzman) [inline app/collections/lane/[slug]/page.tsx:22]: Remove `dc|` from here. The prefix is added automatically in the `trackVirtualPageView` function when it gets sent to Adobe.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Looks good just a minor change for one page name value.

The following is not relevant to the AA update in this PR but it's very apparent from reviewing this:
I just noticed there are two patterns for rendering a page component, and therefore where `PageLayout` is rendered. Either it's in the App Router `page.tsx` file or in a specific `[x]Page.tsx` file. Either is fine but I recommend you and @7emansell  pick one for consistency. It does not need to be part of this PR.

**Review by EdwinGuzman — APPROVED** [review]
The AA chrome extension works but I don't think it picks up virtual views. Is this something you would normally test in the QA env or in the AA dashboard? (AA is picked up in Vercel for the entities dev account)

**Comment by avertrees** [comment]
> The AA chrome extension works but I don't think it picks up virtual views. Is this something you would normally test in the QA env or in the AA dashboard? (AA is picked up in Vercel for the entities dev account)

@EdwinGuzman yes, Lana or Bridgette would check the QA dashboard. It takes a little while for the data to send to the dashboard too.

When you say AA is picked up in Vercel in the entities dev account, do you mean that you're seeing the page names for these pages in the NYPL entities dev account when you preview them in Vercel?

**Comment by EdwinGuzman** [comment]
Yeah, I see the suite is picked up in the chrome extension and in the dashboard I see:
Screenshot 2024-09-03 at 11 14 58 AM

So it's working. You can see some issues like `dc|dc|collections|lane|photographs` but that has been fixed and also the new pages like `dc|search-results`, `dc|all-collections`, etc.

**Comment by avertrees** [comment]
@EdwinGuzman cool, yeah I think this is enough for now. I'll merge this in so Lana and Bridgette can review.

**Pull request merged**